### ISSUE TITLE
RailCraft triggers IC2 reactor loading

### ISSUE THREAD

**Issue by LemADEC** [LemADEC/WarpDrive#95]
As reported by PowerBowser: 

```
Okay, I've narrowed down the problem behind my crash above. I get the crash when I include Railcraft in my mods. Once I narrowed down the crash, I updated RC 9.6.1.0 to 9.7.0.0. I got the same crash, the one mentioning the missing method related to IC2 reactor lasers.

Railcraft has some IC2 integration. There's a steam-powered multiblock that acts as an IC2 generator, and the new electric train system has some integration too, I think.
```

**Comment by LemADEC** [comment]
Can't reproduce with Railcraft + RotaryCraft - IC2.

**Comment by LemADEC** [comment]
Fixed in d9644873cc5cfd5a0ac9ea55a6127b7b4d54ea54
Issue is only reproduced in production environment due to RailCraft only providing a subset of IC2API.
